newrelic_agent.log

```diff
@@ -0,0 +1,68 @@
+{"v":0,"level":30,"name":"newrelic","hostname":"FrankiePC","pid":9408,"time":"2020-12-10T23:35:40.698Z","msg":"Using New Relic for Node.js. Agent version: 7.0.2; Node version: v13.6.0."}
```

Comment:
You want to ignore this file. The logs should be visible locally, and other users who acquire this repository will not need to use these logs. 

**Rationale**: These error logs are mostly used by you to debug, it won't be of much help to the next person who gets it, nor will it be useful if it's displayed on GitHub.